Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -1361,8 +1361,7 @@ def test_module(_: dict, params: dict):
         2. Valid cache_refresh_rate
     """
     if not params.get('longRunningPort'):
-        # This is for the autogeneration port feature before port allocation.
-        params['longRunningPort'] = '1111'
```

Comment:
Let's just revert for this change. so no need to do anything here. 
Also lets make all ports go for this number 1111